designer/server/src/routes/forms/create.js

```diff
@@ -49,16 +74,15 @@ export default [
       yar.clear(sessionNames.validationFailure)
 
       // Redirect to first step
-      return h.redirect('/create/title').temporary()
+      return h.redirect(ROUTE_PATH_CREATE_TITLE).temporary()
     }
   }),
-
```

Comment:
Mind adding these spaces back in?

It was getting a bit cramped so I spaced out the JSDoc comments

Can make it an ESLint rule if it's something we can auto-format next time

Reply:
Done